.github/workflows/ci-cd.yaml

```diff
@@ -256,51 +256,56 @@ jobs:
 
       - name: Checkout
         uses: actions/checkout@v3
-
-      - uses: actions-ecosystem/action-get-latest-tag@v1
-        id: latest-tag
         with:
-          semver_only: true
-          initial_version: 0.0.1
+          fetch-depth: 0
 
       - name: Configure OSC
         # OSC credentials must be configured beforehand as the HOME variables cannot be changed from /github/home
         # that is used to run osc commands
         run: |
-          /scripts/init_osc_creds.sh
           mkdir -p $HOME/.config/osc
-          cp /root/.config/osc/oscrc $HOME/.config/osc
+          cp /home/osc/.config/osc/oscrc $HOME/.config/osc
+          /scripts/init_osc_creds.sh
+
+      - name: Set version
+        run: |
+          git config --global --add safe.directory /__w/runner/runner
+          VERSION=$(./hack/get_version_from_git.sh)
+          sed -i 's~%%REVISION%%~${{ github.sha }}~' $PACKAGE_DIR/_service && \
+          sed -i 's~%%REPOSITORY%%~${{ github.repository }}~' $PACKAGE_DIR/_service && \
+          sed -i 's~%%VERSION%%~'"${VERSION}"'~' $PACKAGE_DIR/_service && \
+          sed -i 's~%%VERSION%%~'"${VERSION}"'~' $PACKAGE_DIR/Dockerfile
+          # "+" character is not allowed in OBS dockerfile version strings
+          VERSION=${VERSION//[+]/-}
+          sed -i 's~%%OBS_VERSION%%~'"${VERSION}"'~' $PACKAGE_DIR/Dockerfile
+
+      - name: create vendor dependencies archive
+        # note the following tar options to strip all the things that could make the archive different without the content actually changing
+        # to make it easier to identify when dependencies changed
+        run: |
+          go mod vendor
+          tar --sort=name --owner=root:0 --group=root:0 --mtime='UTC 1970-01-01' -c vendor | gzip -n > vendor.tar.gz
+
+      - name: checkout and prepare OBS package
+        run: |
+          osc checkout $OBS_PROJECT $PACKAGE_NAME -o $OSC_CHECKOUT_DIR
+          cp $PACKAGE_DIR/_service $OSC_CHECKOUT_DIR
+          rm -v $OSC_CHECKOUT_DIR/*.tar.gz
```

Comment:
I don't think we can do this `rm` here. We would be removing the `ansible-4.6.0.tar.gz ` file. Maybe 

```
rm -v $OSC_CHECKOUT_DIR/runner*.tar.gz
```

Reply:
ohh, good catch, I'll change it so that we only remove the sources, so runner and vendor.